libs/mocks/build.gradle

```diff
@@ -18,11 +18,32 @@ android {
 }
 
 dependencies {
-    implementation("com.github.tomakehurst:wiremock:$wiremockVersion") {
+    api("com.github.tomakehurst:wiremock") {
```

Comment:
Question (❓): Why the `api` change, just so that you could maybe remove the configuration from the main `WordPress` module? 🤔

1. I also saw that the dependency analysis report was suggesting the `api` change, so I am comfortable with it, I just wanted to understand the need, just as an FYI for me.
2. Btw, now that you made this `api` change, and removed the `wiremock` dependency from the main `WordPress` module, the dependency analysis report is asking us to actually declare this transitive dependency directly. If you're curious, just run `./gradlew buildHealth` on this branch and check-it-out for yourself.

Reply:
> Why the api change, just so that you could maybe remove the configuration from the main WordPress module? 🤔

Yes

I understand it'd be nice to add wiremock as direct dependency in `WordPress`, but unfortunately it's not that easy with constraints - they won't be applied in `WordPress` module.

btw: I've tried to run `buildHealth` task, but I was getting Java's heap memory exception. Do you experience similar locally?

Reply:
> I understand it'd be nice to add wiremock as direct dependency in WordPress, but unfortunately it's not that easy with constraints - they won't be applied in WordPress module.

Ok, got it, let's leave it at that for now. 👍 

> btw: I've tried to run buildHealth task, but I was getting Java's heap memory exception. Do you experience similar locally?

Have you applies the new configuration from secrets lately? 😊 

Meaning, do you have the `dependency.analysis.android.ignored.variants=xyz` configuration on your local `gradle.properties` file to exclude all variants and keep just one, there are lots, maybe that's why you get the memory exception! 😅